lib/browser/guest-window-manager.js

```diff
@@ -219,6 +219,8 @@ const canAccessWindow = function (sender, target) {
 
 // Routed window.open messages with raw options
 ipcMainInternal.on('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_OPEN', (event, url, frameName, features) => {
+  // This should only be allowed for senders that have nativeWindowOpen: false
+  if (event.sender.getLastWebPreferences().nativeWindowOpen || event.sender.getLastWebPreferences().sandbox) { throw new Error('GUEST_WINDOW_MANAGER_WINDOW_OPEN denied: expected native window.open'); }
```

Comment:
sandbox: true does not imply nativeWindowOpen afaik

Reply:
it does, since https://github.com/electron/electron/pull/18273